### ISSUE TITLE
Release version 0.6.0!

### ISSUE THREAD

**Issue by Spagy** [Spagy/issue-test#6]
Make sure to tag the release with a proper semantic version i.e. 0.6.0-beta or 0.6.0 rather than beta-0.6.0.

I'm aware that this release will not be a beta version of 0.6.0 but rather that 0.6.0 will itself be a beta version but beta-0.6.0 is not a proper semantic version and so will break the update checking code, which relies on SemVer.

0.6.0 could (should?) be used instead of 0.6.0-beta since the "beta" part is superfluous because the major version is less than 1.

0.6.0-beta would imply that this is only the beta version of 0.6.0 and there will be a full, bug-tested release of 0.6.0 later. Not that I think anyone will misunderstand, it's just that we are using SemVer, so I think we should use it properly.

TL;DR
Could we tag next release as 0.6.0 please?
